### DISCUSSION TITLE
Rol of UV468 array in facemeshcoords.ts

### DISCUSSION THREAD

**Discussion by saeidn95** [vladmandic/human#436] [Q&A]
I would appreciate an explanation to on the role of UV468 in (https://github.com/vladmandic/human/blob/62396317f5c31b2ec5bc7de2e9eb53b8978b9f28/src/face/facemeshcoords.ts#L81C14-L81C20).  The best that I have been able to trace it is to  this line code (https://github.com/vladmandic/human/blob/62396317f5c31b2ec5bc7de2e9eb53b8978b9f28/src/human.ts#L167).  But I cannot figure out where and how it is being used.

**Comment by vladmandic** [comment]
that's a precalculated standardized uvmap for ease-of-use with 3d libraries such as three.js or babylonjs. 
for example, see <https://github.com/vladmandic/human-motion>
